microprofile-config-keyvault/src/main/java/com/azure/microprofile/config/keyvault/NoCacheAzureKeyVaultOperation.java

```diff
@@ -7,22 +7,47 @@
 import java.util.*;
 import java.util.logging.Logger;
 
-
+/**
+ * This class is used to fetch the secrets from Azure Key Vault.
+ *
+ * <ul>
+ *     <li>It does not cache the secrets. It fetches the secrets from Azure Key Vault every time.</li>
+ *     <li>A secret name regular expression is used to filter out invalid secret names to reduce unnecessary calls to Azure Key Vault.</li>
+ *     <li>The default value of the secret name regular expression is "^[0-9a-zA-Z-]+$".</li>
+ * </ul>
+ */
 class NoCacheAzureKeyVaultOperation implements AzureKeyVaultOperation {
+    protected static final String DEFAULT_SECRET_NAME_REGEX = "^[0-9a-zA-Z-]+$";
```

Comment:
Same story here regarding `protected` vs. `private`.

Reply:
Good point, also fixed in [510d4a2](https://github.com/Azure/azure-microprofile/pull/7/commits/510d4a26e21be1606068969e47685f3e0cc7e6be).